relayer/aztec/relayer.go

```diff
@@ -68,19 +83,19 @@ type SpyClient struct {
 
 // NewSpyClient creates a new client for the Wormhole spy service
 func NewSpyClient(endpoint string) (*SpyClient, error) {
-	log.Printf("[SpyClient] Connecting to spy service at %s...", endpoint)
+	logger.Info("Connecting to spy service", zap.String("endpoint", endpoint))
```

Comment:
Maybe add logger to `SpyClient` struct and init it here instead of using global variable?